Xabe.FFmpeg.Test/ProbeTests.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Xabe.FFmpeg.Test
+{
+    public class ProbeTests
```

Comment:
Please add test to catch FFprobeException

Reply:
outdated